shardingsphere-governance/shardingsphere-governance-core/src/main/java/org/apache/shardingsphere/governance/core/registry/RegistryCenter.java

```diff
@@ -148,6 +148,7 @@ public void persistDataSourceConfigurations(final String schemaName, final Map<S
     
     private void addDataSourceConfigurations(final String schemaName, final Map<String, DataSourceConfiguration> dataSourceConfigurations) {
         Map<String, DataSourceConfiguration> dataSourceConfigurationMap = loadDataSourceConfigurations(schemaName);
+        dataSourceConfigurationMap = new LinkedHashMap<>(dataSourceConfigurationMap);
```

Comment:
How about we fix this issue in `convertDataSourceConfigurations()` of `YamlConfigurationConverter`? It looks a little puzzled that `new LinkedHashMap<>(dataSourceConfigurationMap)`.

Reply:
That's ok.